README.md

```diff
@@ -78,7 +67,26 @@ export class Sample {
 
 ```
 
+## Development
```

Comment:
I think instructions on how to run the tests would be helpful

Reply:
> @JGiter One of the tests is failing when I run `npm run test:prod`. Should it be passing?
> 
> ![image](https://user-images.githubusercontent.com/4407464/103704704-f1633000-4f66-11eb-979d-69311398328c.png)

This is strange. I will look at it. Thanks for noticing.

Reply:
@jrhender I can't reproduce this error and `test:prod` is used during CI, which is passes

Reply:
@JGiter I think that the issue is that the `getSubdomains` function is returning instead of continuing if same/less length. Shouldn't line 1045 in the screenshot below be `if (nameArray.length <= role.length) continue;`? The test passes for me if I change it to that. What is happening with `return` is that the function is returning `undefined` because `role` (`['org1', 'root']`) is the same length as `nameArray` (`['root', 'org1']`).
![image](https://user-images.githubusercontent.com/4407464/103791334-90d40180-4fff-11eb-962f-6c0e0a90d7d7.png)

Reply:
This error is fixed in this PR

Reply:
Ah yes, my fault, I just realized that. Thanks